1-Vanilla/index.js

```diff
@@ -0,0 +1,172 @@
+let editedID = 0;
+let editMode = false;
+
+function getNewID(){
+    return Math.floor(Math.random()*1000)
```

Comment:
not good enough. the probability of getting the same id is 1/1000 which is pretty high. why didn't u take the guid generation function?

```
function uuidv4() {
  return ([1e7]+-1e3+-4e3+-8e3+-1e11).replace(/[018]/g, c =>
    (c ^ crypto.getRandomValues(new Uint8Array(1))[0] & 15 >> c / 4).toString(16)
  );
}
```